.circleci/config.yml

```diff
@@ -378,6 +378,24 @@ jobs:
           working_directory: packages/contracts-bedrock
           no_output_timeout: 15m
 
+  contracts-bedrock-static-analysis:
+    docker:
+      - image: <<pipeline.parameters.ci_builder_image>>
+    resource_class: xlarge
+    steps:
+      - checkout
+      - check-changed:
+          patterns: contracts-bedrock,op-node
+      - run:
+          name: print slither version
```

Comment:
What do you think about removing this step and just running the version command in the bash script? Makes it more reproducible locally